__tests__/app.test.js

```diff
@@ -92,3 +92,27 @@ describe("/api/articles/:articles_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test.only("GET 200: return all the articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
```

Comment:
if you use the jest matcher toHaveProperty() it takes to arguments the first is the property name and the seconds you can check the type with the jest matcher expect.any(String) etc, check out the docs